types/index.ts

```diff
@@ -0,0 +1,33 @@
+export type ButtonProps = {
+  className: string;
+  children: any;
+  color: string;
+  icon: any;
+  leftIcon: any;
+  onClick: any;
+  outline: boolean;
+  rightIcon: any;
+  size: string;
+};
+
+export type Sizes = {
+  xs: string;
+  sm: string;
+  md: string;
+  lg: string;
+  xl: string;
+};
+
+type ColorType = {
+  primary: string;
+  secondary: string;
+  "secondary-50": string;
```

Comment:
We still need to update this. We can change this to `light`.

```suggestion
  light: string;
```